Doc/using/cmdline.rst

```diff
@@ -994,3 +994,101 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
+
+The first entry in the module search path is the directory that contains the
+input script, if there is one. Otherwise, the first entry is the current
+directory, which is the case when executing the interactive shell, a ``-c``
+command, or ``-m`` module.
+
+The :envvar:`PYTHONPATH` environment variable is often used to add directories
+to the search path. If this environment variable is found then the contents are
+added to the module search path.
+
+The next items added are the directories containing standard Python modules as
+well as any shared libraries (DLL/dylib/so) that these modules depend on. The
+directory with the platform-independent Python modules is called ``prefix``.
+The directory with the shared libraries is called ``exec_prefix``.
+
+The :envvar:`PYTHONHOME` environment variable may be used to set the ``prefix``
+and ``exec_prefix`` locations. Otherwise these directories are found by using
+the Python executable as a starting point and then looking for various 'landmark'
+files and directories. Note that any symbolic links are followed so the real
+Python executable location is used as the search starting point. On MacOS, the
```

Comment:
MacOS -> macOS

Reply:
But, come to think of it, `PYTHONEXECUTABLE` is a holdover from ancient days. It really should be deprecated and eventually removed.  I've opened [bpo-46901](https://bugs.python.org/issue46901) to do so. In any case, we should not encourage its use so let's remove this sentence.

Reply:
I have made the requested changes; please review again.